### PR TITLE
target dummy

### PR DESCRIPTION
**Pull request by Tarmunora**
[QOL]
## About the PR 
Adds a testing dummy for testing stuff for coding. It gets the assday damage popups and a verb to fullheal it (in case someone wants to add these to murderbox or something)
This time working from a clean master instead of other code :shcoderfrog:

## Why's this needed? 
Makes it a bit easier to test damaging things without having to check health all the time

### CODE REVIEW

**Comment by ZeWaka** [comment]
I think an addition to the code guide document might be appropriate for this, in `Useful Things`

**Comment by Tarmunora** [comment]
aaaa fuck

File: code/mob/living/carbon/human/procs/Life.dm [hunk 1]
```diff
@@ -478,6 +478,9 @@
 
 #if ASS_JAM //Oh neat apparently this has to do with cool maptext for your health, very neat. plz comment cool things like this so I know what all is on assjam!
 	src.UpdateDamage()
+#else
+	if(istype(src, /mob/living/carbon/human/dummy)) //do it for target dummies all the time
+		src.UpdateDamage()
```
Comment (flrsh) [inline code/mob/living/carbon/human/procs/Life.dm:483]: I'd rather this just have

```
/mob/living/carbon/human/dummy/Life(datum/controller/process/mobs/parent)
  if (..(parent))
    return 1
  src.UpdateDamage()
```

You could put this bit right after your `/mob/living/carbon/human/dummy/UpdateDamage()` in `update_icon.dm` so it wouldn't double call on Ass Day.

**Comment by ZeWaka** [comment]
If you want to fix travis failing, you'll have to pull from master.

**Pull request merged**